x/ccv/provider/keeper/proposal_test.go

```diff
@@ -1045,8 +1045,14 @@ func TestHandleEquivocationProposal(t *testing.T) {
 	propPass := &providertypes.EquivocationProposal{
 		Equivocations: []*evidencetypes.Equivocation{tcPass[0], tcPass[1]},
 	}
-	keeper.SetSlashLog(ctx, tcPass[0].GetConsensusAddress())
```

Comment:
Previously, `tcPass[0].GetConsensusAddress()` would return nil here. This caused odd behavior in `HandleEquivocationProposal`